docs/source/create/avatars/create-avatars.rst

```diff
@@ -2,13 +2,13 @@
 Create Your Own Avatar
 ##############################
 
-There are three ways to create your own avatar. You can either:
+There are two ways to create your own avatar. You can either:
```

Comment:
There are three items in the list, not two. Perhaps better wording would be to use "get" instead of "create". And the "either" doesn't make sense either, given that there are more than two items in the list.

Perhaps better wording would be:"
```
There are three ways to get your own avatar:
```